src/globus_sdk/services/timer/errors.py

```diff
@@ -0,0 +1,60 @@
+from typing import Any, Dict, List
+
+from globus_sdk.exc import GlobusAPIError
+
+
+class TimerAPIError(GlobusAPIError):
+    """
+    Error class to represent error responses from Timer.
+
+    Has no particular additions to the base ``GlobusAPIError``, but implements a
+    different method for parsing error responses from Timer due to the differences
+    between pydantic-generated errors and other errors.
+    """
+
+    MESSAGE_FIELDS: List[str] = []  # we are overriding `_load_from_json` instead
```

Comment:
I don't think you need to set this classvar at all. It's not necessary to remove it, but if you want to drop a couple of lines I think it's fine to omit.

Reply:
Hmm, sure. I wrote this in partly to indicate that the attr on the base class would not be correct. This version seems slightly more correct to me in that respect (even if it isn't used)? I would set it to `None` but that would be a type error.

Reply:
Yeah, I thought that might be the reasoning here -- it's not used but this is the only way to write "there are no `MESSAGE_FIELDS`". Makes sense. Let's mark this resolved; I'm good with keeping it.

I just wanted to make sure you knew it's not technically necessary.